src/components/UI/Card/CartCards/CartCards.jsx

```diff
@@ -1,88 +1,137 @@
+import IcRoundMinus from '../../Icons/IcRoundMinus';
+import IcRoundPlus from '../../Icons/IcRoundPlus';
+import "./CartCards.css"
+import IcRoundWishlist from '../../Icons/IcRoundWishlist';
 function CartCards(props) {
-    const { _id, title ,productImage , author,price,discount,discountedPrice } =props;
-    const {cartState,setCartState}= useCart();
-    const { WishListState, setWishListState } = useWishList();
-    
-    const RemoveItemsFromCartHandler=async(id)=>{
-        try{
+    const {
+        _id,
+        title,
+        productImage,
+        author,
+        price,
+        discount,
+        discountedPrice,
+        quantity
+    } = props;
+    const {cartState, setCartState} = useCart();
+    const {WishListState, setWishListState} = useWishList();
+
+    const RemoveItemsFromCartHandler = async (id) => {
+        try {
             console.log(id);
-            await axios.delete(`/api/user/cart/${id}`,
-                    { headers:{authorization:localStorage.getItem("feetz")}})
-                        .then((res)=>{
-                            console.log(res);
-                            if(res.status===200){
-                                setCartState(res.data.cart);
-                            }
-                        })
-                        .catch((error)=>{
-                            console.log(error.message);
-                        });
-        }
-        catch(err){
-            console.log("error ",err.message);
+            await axios.delete(`/api/user/cart/${id}`, {
+                headers: {
+                    authorization: localStorage.getItem("feetz")
+                }
+            }).then((res) => {
+                console.log(res);
+                if (res.status === 200) {
+                    setCartState(res.data.cart);
+                }
+            }).catch((error) => {
+                console.log(error.message);
+            });
+        } catch (err) {
+            console.log("error ", err.message);
         }
     }
 
-    const addToWishlistHandler = async (item)=>{
-        try{
+    const addToWishlistHandler = async (item) => {
+        try {
             console.log(item);
-            await axios.post("/api/user/wishlist",
-                    { "product":item},
-                    { headers:{authorization:localStorage.getItem("feetz")}})
-                        .then((res)=>{
-                            console.log(res);
-                            if(res.status===201){
-                                setWishListState(res.data.wishlist);
-                            }
-                        })
-                        .catch((error)=>{
-                            console.log(error.message);
-                        });
+            await axios.post("/api/user/wishlist", {
+                "product": item
+            }, {
+                headers: {
+                    authorization: localStorage.getItem("feetz")
```

Comment:
You can name this property to something more meaningful like user.

Reply:
yeah made constant and made them meaningfull